tests/pr_test.go

```diff
@@ -20,28 +19,27 @@ func setupOptions(t *testing.T, prefix string) *testhelper.TestOptions {
 	})
 
 	options.TerraformVars = map[string]interface{}{
-		"service_name":    fmt.Sprintf("%s-%s", options.Prefix, "cis-new"),
-		"pool_name":       fmt.Sprintf("%s-%s", options.Prefix, "cis-pool"),
-		"steering_policy": "off",
+		"prefix":      options.Prefix,
+		"domain_name": domain_name,
 	}
 
 	return options
 }
 
-func TestRunBasicExample(t *testing.T) {
+func TestRunCompleteExample(t *testing.T) {
 	t.Parallel()
 
-	options := setupOptions(t, "cis-new")
+	options := setupOptions(t, "cis-new", "prtest.goldeneye.dev.cloud.ibm.com")
```

Comment:
Instead of set the domain directly in the setupOptions call, maybe would be better to set it in a variable, this would allow value duplication for the TestRunUpgradeCompleteExample